### PR TITLE
Added new question about agile methods

### PR DESCRIPTION
**Pull request by FranciscoHeronildo**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/agile-quiz.ts [hunk 1]
```diff
@@ -164,6 +164,17 @@ const agileQuiz = [
       "Points estimate the amount of effort required to deliver a user story and are assigned using the Fibonacci sequence",
     Link: "https://www.freecodecamp.org/news/complete-guide-to-agile-methodology/",
   },
+  {
+    Question:
+      "In Agile, what is it used for in Planning Poker?",
```
Comment (jdwilkin4) [inline src/data/agile-quiz.ts:169]:
```suggestion
      "In Agile, what is Planning Poker?",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/agile-quiz.ts (hunk 1 above)
Comment (FranciscoHeronildo) [reply src/data/agile-quiz.ts]: made the changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**